pyccel/ast/core.py

```diff
@@ -3827,10 +3827,9 @@ class Assert(Basic):
     _attribute_nodes = ('_test',)
     #TODO add type check in the semantic stage
```

Comment:
Is this TODO comment still relevant?

Reply:
@Pinkyboi @EmilyBourne

Reply:
It is no longer relevant as the argument is now wrapped in a `PythonBool` if it has the wrong type (a few lines after this comment)

Reply:
Then I am going to remove it, and merge this PR.